src/components/ImageAsset.vue

```diff
@@ -12,7 +12,8 @@
   <img
     v-if="fallbackImageSrcSet"
     class="fallback"
-    :alt="`${alt} Image failed to load.`"
+    title="Image failed to load."
```

Comment:
I'm not sure which guidelines we follow exactly for DocC Render, but generally tooltips on macOS do not end with a period: https://developer.apple.com/design/human-interface-guidelines/macos/user-interaction/help/. However on Windows, it seems like they do: https://docs.microsoft.com/en-us/windows/win32/controls/tooltip-controls, so there isn't a one-size-fits-all solution. cc @willimholte

Reply:
Fair point, we should cut the period.